### PR TITLE
fix: max width of editor container on mobile

### PR DESCRIPTION
**Pull request by luka-nextcloud**
### 📝 Summary

* Resolves: #3844

#### 🖼️ Screenshots

🏚️ Before | 🏡 After
---|---
image | image

### 🚧 TODO

- [ ] ...

### 🏁 Checklist

- [ ] Code is properly formatted (`npm run lint` / `npm run stylelint` / `composer run cs:check`)
- [ ] [Sign-off message](https://probot.github.io/apps/dco/) is added to all commits
- [ ] [Tests](https://github.com/nextcloud/text#-testing-the-app) (unit, integration and/or end-to-end) passing and the changes are covered with tests
- [ ] Documentation ([README](https://github.com/nextcloud/text/blob/main/README.md) or [documentation](https://github.com/nextcloud/documentation/blob/master/admin_manual/configuration_server/text_configuration.rst#L2)) has been updated or is not required

### CODE REVIEW

File: src/components/Editor/ContentContainer.vue [hunk 1]
```diff
@@ -67,6 +67,12 @@ export default {
 		width: 100%;
 	}
 
+	@media (max-width: 670px) {
```
Comment (juliushaertl) [inline src/components/Editor/ContentContainer.vue:70]: It wasn't immediately obvious to me that this is the value of `--text-editor-max-width`, maybe add a code comment to this.

Maybe we can just set a max-width in the above general `.editor_content` rule like this to keep it simple?

```css
.editor__content {
	max-width: min(var(--text-editor-max-width, calc(100vw - 16px));
```

**Review by juliushaertl — CHANGES_REQUESTED** [review]

File: src/components/Editor/ContentContainer.vue (hunk 1 above)
Comment (luka-nextcloud) [reply src/components/Editor/ContentContainer.vue]: Ok, please check again.

**Review by juliushaertl — APPROVED** [review]

**Comment by juliushaertl** [comment]
/backport 8765b4f8ef1dd6a7ded01b357d31f8e9a8c9328d to stable27

**Comment by juliushaertl** [comment]
/backport 8765b4f8ef1dd6a7ded01b357d31f8e9a8c9328d to stable26

**Pull request merged**